server/http/request.go

```diff
@@ -24,7 +24,7 @@ func announceRequest(r *http.Request, cfg *httpConfig) (*chihaya.AnnounceRequest
 
 	eventStr, err := q.String("event")
 	if err != nil {
-		return nil, tracker.ClientError("failed to parse parameter: event")
+		eventStr = "none"
```

Comment:
maybe we should check if `err == query.ErrKeyNotFound`? It's the only possible error returned right now, but that could change in the future.

@jzelinskie ?